src/styles/index.css

```diff
@@ -0,0 +1,117 @@
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: #060b35;
```

Comment:
Move all repeatable colosrs to CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

Reply:
Check all other similar cases.

Reply:
Not resolved.